### PR TITLE
fix: Update Arq42 and DockerFile Scripts

### PR DESCRIPTION
**Pull request by fabiodmota**
## Description
- Switched the shell interpreter from `bash` to `sh` in our scripts to enhance portability and reduce dependencies. This change helps ensure our scripts run reliably in environments where `bash` may not be available. All relevant scripts have been updated and tested to confirm compatibility with `sh`.
- Updated arq42 documentation

Fixed for issues:
Arq42 docs:  https://github.com/eclipse-tractusx/vas-country-risk/issues/40
Script change: https://github.com/eclipse-tractusx/vas-country-risk/issues/22

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by fabiodmota** [comment]
Please  review on topics and leave comment
@enzio-bot  on Arq42 documentation
@alexsilva-CGI on script change

**Review by alexsilva-CGI — APPROVED** [review]
Looks good to me

File: docs/Images/reportUpdate.png
[the remark this answers is not in the document]
Comment (fabiodmota) [reply docs/Images/reportUpdate.png:1]: I didn't understand; most of the images are referenced with just "image" and a date. Most of them were included in some documentation. What do you really suggest in this case?

File: docs/Arc42-Documentation.md [hunk 1]
```diff
@@ -268,12 +277,23 @@ For this component, it is necessary to call three different APIs. The APIs used
...
 
+## Dashboard Controller
+
+![apiUpload](../docs/Images/reportUpdate.png)
+
+This endpoint is used to update the report. Let's say a user has already saved their report; when updating it, such as changing the selected range to color in different ways, just press the save symbol and the new settings will be updated.
+
+![updateReport](../docs/Images/updateReport.png)
+
+![saveChangesReport](../docs/Images/saveChangesReport.png)
```
[the remark this answers is not in the document]
Comment (fabiodmota) [reply docs/Arc42-Documentation.md:294]: No , the two images are from the same action , one is the first user action the second is a popup for changes 
https://github.com/catenax-ng/tx-vas-country-risk-frontend/blob/main/docs/Arc42-Documentation.md#dashboard-controller

File: Dockerfile [hunk 2]
```diff
@@ -67,9 +67,9 @@ COPY ./scripts/inject-dynamic-env.sh /docker-entrypoint.d/00-inject-dynamic-env.
 
 RUN chmod +x /docker-entrypoint.d/00-inject-dynamic-env.sh
 
-# Install bash and update vulnerable packages
+# Update vulnerable packages
 RUN apk update && \
-    apk add --no-cache bash
+    apk upgrade
```
Comment (almadigabor) [inline Dockerfile:72]: `apk upgrade` or any other package managed upgrades are not allowed. The reason is that these official base images are scanned by Docker Hub and the results are published afterwards. Changing the image this way would mean those scan results are no longer valid. If you have any issues with the current image, consider upgrading to a newer tag.

**Review by almadigabor — CHANGES_REQUESTED** [review]

**Review by almadigabor — APPROVED** [review]

**Pull request merged**